app/controllers/orders/conversation.js

```diff
@@ -75,6 +76,11 @@ export default detail.extend({
 
       // Animate and scroll to bottom
       this.autoScroll();
+    },
+    markRead(){
+      this.get("sortedMessages")
+        .filterBy('state', 'unread')
+        .forEach(m => this.get("messagesUtil").markRead(m));
```

Comment:
use more expressive variable name `m` doesn't say anything.

Reply:
sure, i'll name it `message`